services/controlplane/client.go

```diff
@@ -111,16 +131,16 @@ func (c *Client) retry(ctx context.Context, fn func() error) error {
 	return backoff.Retry(fn, opts)
 }
 
-func (c *Client) GetDestinationSSHKeyPair(ctx context.Context, destID string) (kp SSHKeyPair, err error) {
```

Comment:
Some tests seem to fail because of this change

Reply:
Yeah I now sorry I was stuck in a call, will fix it asap thanks @cisse21